src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+export default function Button(props) {
+  const { buttonName } = props;
+  return (
+    <div>
+      <button type="button">
+        {buttonName}
+      </button>
+    </div>
```

Comment:
- [ ] [REQUIRED] Please don't wrap `button` inside div because, when I open your APP on my local, the result looks like 
 :arrow_down:
![1](https://user-images.githubusercontent.com/58552748/129173655-c76e3edc-4cf6-42c4-8ca7-9c7fc0269655.gif)